### PR TITLE
Validation

### PR DESCRIPTION
**Pull request by CananOzyol**

### CODE REVIEW

File: frontend/src/components/RegisterForm.tsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    const [showPassword, setShowPassword] = useState<boolean>(false);
+    const [errorUsername, setErrorUsername] = useState<string>("");
+    const [errorPassword, setErrorPassword] = useState<string>("");
+
+    const navigate = useNavigate()
+
+    function handleSubmit(event: FormEvent) {
+        event.preventDefault()
+        props.onRegister(username, password)
+        props.onLogin(username, password);
```
Comment (Satttoshi) [inline frontend/src/components/RegisterForm.tsx:34]: this looks cool, but does this work? I mean the user is being registered and then being logged in, is the Data fast enough written into the DB (considering the case when its online aswell), it seems like there is something to await first. Not sure tho

**Review by Satttoshi — APPROVED** [review]
Naise, this looks so beautiful my Eyes are in tears 

LGTM :rocket:

File: frontend/src/App.tsx [hunk 2]
```diff
@@ -30,8 +31,18 @@ export default function App() {
         me();
     }, [user])
 
+    useEffect(() => {
+        axios.get("/api/quiz")
+            .then(response => response.data)
+            .catch(console.error)
+            .then(data => {
+                console.log(data)
```
[the remark this answers is not in the document]
Comment (toshydev) [reply frontend/src/App.tsx:39]: so sorry officer!!!

File: backend/src/main/java/org/partypets/backend/model/DTOParty.java [hunk 3]
```diff
@@ -1,16 +1,26 @@
...
 
-import java.util.Date;
+import java.time.LocalDate;
 
 @Data
 @NoArgsConstructor
 @AllArgsConstructor
 public class DTOParty {
-    private Date date;
+    @NotNull
```
[the remark this answers is not in the document]
Comment (toshydev) [reply backend/src/main/java/org/partypets/backend/model/DTOParty.java:17]: Copy that! will do!

**Pull request merged**